runtime/pom.xml

```diff
@@ -73,6 +72,7 @@
             <goals>
               <goal>resource</goal>
               <goal>helm</goal>
+              <goal>build</goal>
```

Comment:
It's foobarred - no hook gets sent it seems. Enabling the project from circleci doesn't actually add the webhook either...